haskell/nixpkgs.bzl

```diff
@@ -173,10 +176,15 @@ def haskell_register_ghc_nixpkgs(
         locale = None,
         repositories = {},
         repository = None,
-        nix_file_content = None):
+        nix_file_content = None,
+        use_worker = False):
     """Register a package from Nixpkgs as a toolchain.
 
-    Toolchains can be used to compile Haskell code. To have this
+    Args:
+      use_worker: This is a part of experimental support for the persistent worker mode.
+        It is not intended for production usage, yet.
+
+      Toolchains can be used to compile Haskell code. To have this
```

Comment:
```suggestion
    Toolchains can be used to compile Haskell code. To have this
```

Reply:
Wow, @aherrmann, I missed your comment somehow, I'm terribly sorry! I opened #1044 to fix this.